canonical/ubuntu.com
# Create mediatek contact-us form

## Done
*In searching around for how static forms were handled in other places, i stumbled across a number of other smaller issue that i have covered and detailed below*

- Create mediatek contact-us form based on [the copydoc](https://docs.google.com/document/d/1WnWVmVuqd1G_BCA060K1LozS2xYFCDli90sGc1SU7ug/edit#heading=h.dxn10eelvkoc)
- Create js function to parse the extra fields and add them to a hidded field 'Comments_from_lead_c'. This is already being done in dynamic-form.js but only applies to interactive forms. And include a js to handle 'Other' check boxes, an example of the HTML needed is found below
- Removes code from [/embedding/index.html](https://github.com/canonical/ubuntu.com/pull/12612/files#diff-56ddfc54a339d2672c4a98dfe78fc989814a78f58941b9ea42156184b27375a8) that didn't appear to do anything.
- Removed some random console.logs
- White space formatting on a file I check out
- Removes unused fips beta form


HTML for other checkbox example:
```
 <div class="js-other-container">
    <label class="p-checkbox">
      <input class="p-checkbox__input js-other-container__other-toggle" type="checkbox" aria-labelledby="priority-workloads-other">
      <span class="p-checkbox__label" id="priority-workloads-other">Other</span>
    </label>
    <input type="text" id="priority-workloads-other-specified" class="js-other-container__input" placeholder="Please specify" style="opacity: 0; margin-top: .25rem;" aria-label="Other architecture">
  </div>
```
## QA

- Go to https://ubuntu-com-12612.demos.haus/download/mediatek/contact-us and sewe that the form matches the copydoc
- Fill in the form and check the payload contains all information
- Check the formid is correct
- Check it returns to a generic thank-you page
- Check the form is recieved on marketo

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-2250
